physlr/physlr.py

```diff
@@ -1726,19 +1726,32 @@ def subgraph_stats_process(u):
         return Physlr.subgraph_stats(Physlr.graph, u)
 
     def physlr_subgraphs_stats(self):
-        "Retrieve subgraphs' stats."
-        gin = self.read_graph(self.args.FILES)
-        Physlr.filter_edges(gin, self.args.m)
-        print(
-            int(timeit.default_timer() - t0),
-            "Computing statistics of the subgraphs...", file=sys.stderr)
+        "Retrieve statistics of subgraphs."
+        nodes_of_interest = []
+        if len(self.args.FILES) > 1:
+            gin = self.read_graph([self.args.FILES[0]])
+            with open(self.args.FILES[1]) as fin:
+                for line in fin:
+                    nodes_of_interest.append(line.split()[0])
+            print(
+                int(timeit.default_timer() - t0),
+                "Computing statistics for subgraphs of interest...",
+                file=sys.stderr)
+        else:
+            gin = self.read_graph(self.args.FILES)
+            print(
+                int(timeit.default_timer() - t0),
+                "Computing statistics of all subgraphs...", file=sys.stderr)
+            nodes_of_interest = gin
+        #Physlr.filter_edges(gin, self.args.n)
```

Comment:
remove this

Reply:
Why? the function there is file indication a subset of barcodes (vertices), this function reads in those vertices and considers only those vertices for stat retrieval. but if there's no input file for that, it works on the whole graph..

Reply:
I meant remove the commented out line

Reply:
I see :)) I'll do it.

Reply:
done